db/seeds.rb

```diff
@@ -7,3 +7,6 @@
 broc.image.attach(io: File.open(Rails.root.join('app', 'assets', 'images', 'broccoli.png')), filename: 'broccoli.png', content_type: 'image/png')
 potato = Award.find_or_create_by(title: "Breezy", value: 40)
 potato.image.attach(io: File.open(Rails.root.join('app', 'assets', 'images', 'potato.png')), filename: 'potato.png', content_type: 'image/png')
+
+# Admin user
+User.create(email: "admin@admin.com", username: "admin", password: "123456", password_confirmation: "123456", confirmed_at: Time.now, admin: true)
```

Comment:
find_or_create_by?

Reply:
pievienoju if condition, bet ar find_or_create_by nestrada, del password parametra.( pierakstiju more in depth #85 )